moto/elb/responses.py

```diff
@@ -177,6 +183,101 @@ def parse_attribute(attribute_name):
+
+        other_policy = OtherPolicy()
+        policy_name = [value[0] for key, value in self.querystring.items() if "PolicyName" in key][0]
+        other_policy.policy_name = policy_name
+
+        self.elb_backend.create_lb_other_policy(load_balancer_name, other_policy)
+
+        template = self.response_template(CREATE_LOAD_BALANCER_POLICY_TEMPLATE)
+        return template.render()
+
+    def create_app_cookie_stickiness_policy(self):
+        load_balancer_name = self.querystring.get('LoadBalancerName')[0]
+        load_balancer = self.elb_backend.describe_load_balancers(load_balancer_name)[0]
+
+        policy = AppCookieStickinessPolicy()
+        policy_name = [value[0] for key, value in self.querystring.items() if "PolicyName" in key][0]
+        policy.policy_name = policy_name
+        cookie_name = [value[0] for key, value in self.querystring.items() if "CookieName" in key][0]
+        policy.cookie_name = cookie_name
+
+        self.elb_backend.create_app_cookie_stickiness_policy(load_balancer_name, policy)
+
+        template = self.response_template(CREATE_LOAD_BALANCER_POLICY_TEMPLATE)
+        return template.render()
+
+    def create_lbcookie_stickiness_policy(self):
+        load_balancer_name = self.querystring.get('LoadBalancerName')[0]
+        load_balancer = self.elb_backend.describe_load_balancers(load_balancer_name)[0]
+
+        policy = AppCookieStickinessPolicy()
+        policy_name = [value[0] for key, value in self.querystring.items() if "PolicyName" in key][0]
+        policy.policy_name = policy_name
+        cookie_expirations = [value[0] for key, value in self.querystring.items() if "CookieExpirationPeriod" in key]
+        if cookie_expirations:
+            policy.cookie_expiration_period = long(cookie_expirations[0])
+        else:
+            policy.cookie_expiration_period = None
+
+        self.elb_backend.create_lb_cookie_stickiness_policy(load_balancer_name, policy)
+
+        template = self.response_template(CREATE_LOAD_BALANCER_POLICY_TEMPLATE)
+        return template.render()
+
+    def set_load_balancer_policies_of_listener(self):
+        load_balancer_name = self.querystring.get('LoadBalancerName')[0]
+        load_balancer = self.elb_backend.describe_load_balancers(load_balancer_name)[0]
+        load_balancer_port = int(self.querystring.get('LoadBalancerPort')[0])
+
+        mb_listener = [l for l in load_balancer.listeners if int(l.load_balancer_port) == load_balancer_port]
+        if mb_listener:
+            policies = []
+            policy_index = 1
+            while True:
+                try:
+                    policy = self.querystring['PolicyNames.member.{0}'.format(policy_index)][0]
+                except KeyError:
+                    break
+
+                policy_index += 1
+                policies.append(str(policy))
+
+            self.elb_backend.set_load_balancer_policies_of_listener(load_balancer_name, load_balancer_port, policies)
+        # else: explode?
```

Comment:
Silently failing didn't quite seem like the best approach for not finding a load balancer, but I wasn't sure as to what would be better.